sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/TestMethodShouldContainAssertion.MsTest.cs

```diff
@@ -115,11 +115,138 @@ public void ShouldSomething()
             Assert.IsTrue(true);
         }
 
+        public void ExpectSomething()
+        {
+            Assert.IsTrue(true);
+        }
+
+        public void MustSomething()
+        {
+            Assert.IsTrue(true);
+        }
+
+        public void VerifySomething()
+        {
+            Assert.IsTrue(true);
+        }
+
+        public void ValidateSomething()
+        {
+            Assert.IsTrue(true);
+        }
+    }
+
+    [TestClass]
+    public class Program_DataTestMethod
```

Comment:
The rule wasn't handling _DataTestMethod_ so I duplicate all of the existing test cases for that attribute too.

Originally, I wasn't going to duplicate all of the tests, just enough to validate that _DataTestMethod_ is recognized as a test attribute. However, the test cases for the other test rules appear to duplicate all of the cases for each attribute, so I followed that pattern.